src/controller/CHIPDeviceController.cpp

```diff
@@ -1713,17 +1750,17 @@ void DeviceCommissioner::OnDisarmFailsafe(void * context,
 {
     ChipLogProgress(Controller, "Failsafe disarmed");
     DeviceCommissioner * commissioner = static_cast<DeviceCommissioner *>(context);
-    commissioner->DisarmDone();
+    commissioner->CleanupAfterErrorDone();
 }
 
 void DeviceCommissioner::OnDisarmFailsafeFailure(void * context, CHIP_ERROR error)
 {
     ChipLogProgress(Controller, "Ignoring failure to disarm failsafe: %" CHIP_ERROR_FORMAT, error.Format());
     DeviceCommissioner * commissioner = static_cast<DeviceCommissioner *>(context);
-    commissioner->DisarmDone();
+    commissioner->CleanupAfterErrorDone();
 }
 
-void DeviceCommissioner::DisarmDone()
+void DeviceCommissioner::CleanupAfterErrorDone()
```

Comment:
Maybe call this CleanupCompletedAfterError?  If the intent is to say that this is called when we hit an error, do some cleanup, then call this function.